backend/src/akvo/flow_services/error.clj

```diff
@@ -25,7 +26,7 @@
   `(try
      ~body
      (catch Exception e#
-       (.printStackTrace e#)
+       (log/error e#)
```

Comment:
It feels weird that a function that tries to make the error handling more FP has a side effect on it. 

Apart, if there is an error when calling Flow for the notification, like in https://github.com/akvo/akvo-flow-services/blob/500c8b077013a3ded88dbe95e295569e223fe1fc/backend/src/akvo/flow_services/scheduler.clj#L116 then we are going to end up logging the exception twice, once is the log line of this change and another one due to https://github.com/akvo/akvo-flow-services/blob/cd2b30caaf34e2db458e46fbb1aaf2ec30372c65/backend/src/akvo/flow_services/core.clj#L130

A suggestion would to log the error on the "non-pure-FP" side at
https://github.com/akvo/akvo-flow-services/blob/500c8b077013a3ded88dbe95e295569e223fe1fc/backend/src/akvo/flow_services/scheduler.clj#L117
by changing that `(when (e/if-ok...` to `(e/fmap...` and use `timbre/error` in the fn-error param